kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/model/TopicPartitionOffset.scala

```diff
@@ -24,6 +24,9 @@ case class Topic(value: String) extends AnyVal {
 
 object Offset {
 
+  // Define an implicit conversion from Long to Offset
+  implicit def longToOffset(value: Long): Offset = Offset(value)
```

Comment:
This is not advisable as a conversion. Offset itself it's a code syntactic sugar around Long. It's advisable to do offset(value) ad-hoc. The compiler, won't actually create a runtime instance of Offset

Reply:
OK, I got rid of the conversion and added an additional method

```
  def atOffset(offset: Long): TopicPartitionOffset = withOffset(Offset(offset))
```